src/datasets/loaders/10x_datasets/config.vsh.yaml

```diff
@@ -0,0 +1,96 @@
+functionality:
+  name: 10x_datasets
+  namespace: datasets/loaders
+  description: |
+    Download a SpaceRanger h5 gene expression file and spatial imaging data from the 10x genomics website (or someplace else).
+  
+  argument_groups: 
+    - name: Inputs
+      arguments:
+        - name: "--input_expression"
+          type: string
```

Comment:
Make this a `type: file`. If we provide a url to the param_list of a nextflow component, nextflow will automatically download and cache the input files, which is more efficient